src/linagora.esn.unifiedinbox/app/components/sidebar/mailbox-display-tree/mailbox-display-tree.spec.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
```

Comment:
I know a good jmap is server not supposed to do that but hey, software development is messy, especially when a cache is around...

Have you tested how it behaves with an actal loop?

Mbx a have parent id mbx b
And mbx b have parent id mbx a?

Reply:
Interessing, I can test but what is the expected result : control it front side or backend side ?

Reply:
Control backend side,

but resiliency on the front end side.

Just test that it don't create an infinite loop or something nasty IMO